commit 9ee03bfd23709baa77202534f8abe7e66bb160f6
Author: Thomas J. Moore <darktjm@gmail.com>
Date:   Sun Nov 15 19:44:13 2020 -0600

    Skip fetching DLC info during first pass of update-cache, as that info
      is downloaded (again) during 2nd pass, anyway

diff --git a/src/website.cpp b/src/website.cpp
index 5cdaf16..7cc7dab 100644
--- a/src/website.cpp
+++ b/src/website.cpp
@@ -228,7 +228,7 @@ std::vector<gameItem> Website::getGames()
                         }
                     }
 
-                    if (bDownloadDLCInfo)
+                    if (bDownloadDLCInfo && !Globals::globalConfig.bUpdateCache)
                     {
                         game.gamedetailsjson = this->getGameDetailsJSON(game.id);
                         if (!game.gamedetailsjson.empty())

